datalad/support/third/loris_token_generator.py

```diff
@@ -9,10 +9,11 @@
 import sys
 import json
 
-if sys.version_info[0] == 2:
-    import urllib2 as urllib_request
-else:
-    from urllib import request as urllib_request
+import six.moves.urllib
```

Comment:
from six.moves import urllib
?
Since using urllib below

Reply:
I think I've finally got it working with both Python 2 and 3. It's `six.moves.urllib.request` that I needed, not `six.moves.urllib`